ui/app/styles/components/gutter.scss

```diff
@@ -1,4 +1,76 @@
 .gutter {
   height: 100%;
   border-right: 1px solid $grey-blue;
+  overflow: hidden;
+
+  @media #{$mq-hidden-gutter} {
+    border-right: none;
+
+    &.is-open {
+      box-shadow: 0 0 30px darken($nomad-green-darker, 20%);
+    }
+  }
+
+  .collapsed-menu {
+    display: none;
+    position: relative;
+    height: 3.5rem;
+
+    width: $gutter-width;
+    transform: translateX($gutter-width);
+    z-index: $z-gutter;
+
```

Comment:
Are these spaces intentional?

Reply:
Yes and no? I remember thinking it made the code read better, particularly the part where the display is `none` despite all those other styles being there. 

Looking at it again now, I don't find it helpful. Will remove.